docs/installation_guide.rst

```diff
@@ -166,6 +166,49 @@ At this point the rotki docker container should be running and you
 should be able to access rotki frontend, open your browser and go to
 it at ``http://localhost:8084``. You should be able to see the rotki login screen.
 
+
+Configuring backend in docker
+-------------------------------
+It is possible to change the configuration of the backend in the docker container. You can do so
+using two different approaches.
+
+The first approach include mounting a config volume::
+
+   docker run -d --name rotki \
+       -p 8084:80 \
+       -v $HOME/.rotki/data:/data \
+       -v $HOME/.rotki/logs:/logs \
+       -v $HOME/.rotki/config:/config \
+       rotki/rotki:latest
+
+You need to create a ``rotki_config.json`` file and put it inside ``$HOME/.rotki/config`` directory.
+
+.. code-block:: javascript
+
+    {
+       "loglevel": "info",
+       "logfromothermodules": true,
+       "sleep-secs": 22,
+       "max_size_in_mb_all_logs": 550,
+       "max_logfiles_num": 3
+    }
```

Comment:
I guess this works as the list of things that can be modified. Maybe we can add a note saying that all the options are represented above?

Reply:
So the comment bellow is not sufficient, sure I will reword